plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -132,13 +158,14 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
                 "as the native image build process is memory intensive."
     )
 
-    private fun getDockerRunArgs(params: List<String>): List<String> {
+    private fun getDockerRunArgs(params: List<String>, dockerWorkdirPath: String?): List<String> {
```

Comment:
Not sure if this mechanism of passing `dockerWorkdirPath` can be improved or even simplified. Will double check on Monday